From: Otavio Salvador <otavio@ossystems.com.br>
Subject: [PATCH 2/2] [MX23] Do not use safe write when building for MX23 arch

The change 105ae4a60 include a safe write support for use with MX23
but does not include a dummy implementation for other architecture
making build to fail. This add a dummy version to allow the use with
MX23 arch.

Upstream-Status: Pending

Signed-off-by: Otavio Salvador <otavio@ossystems.com.br>
---
 arch/arm/plat-mxs/usb_common.c |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/plat-mxs/usb_common.c b/arch/arm/plat-mxs/usb_common.c
index 700d168..c3bc9a9 100644
--- a/arch/arm/plat-mxs/usb_common.c
+++ b/arch/arm/plat-mxs/usb_common.c
@@ -53,6 +53,8 @@ static void fsl_safe_writel(u32 val32, volatile u32 *addr)
 {
 	__asm__ ("swp %0, %0, [%1]" : : "r"(val32), "r"(addr));
 }
+#else
+static void fsl_safe_writel(u32 val32, volatile u32 *addr) { }
 #endif
 
 void fsl_usb_xcvr_register(struct fsl_xcvr_ops *xcvr_ops)
-- 
1.7.10.4

